core/src/test/scala/unit/kafka/coordinator/group/CoordinatorPartitionWriterTest.scala

```diff
@@ -84,98 +67,28 @@ class CoordinatorPartitionWriterTest {
   }
 
   @Test
-  def testWriteRecords(): Unit = {
```

Comment:
Do we have an equivalent test for the writing of the records in CoordinatorRuntimeTest? I didn't really notice new tests, but saw we have some of the builder logic there. Is it tested by checking equality between the records generated by the helper methods and the output from running the CoordinatorRuntime code?

Reply:
Right. We have many tests in CoordinatorRuntimeTest doing writes. As we fully validate the records now, they cover this.